### DISCUSSION TITLE
Container Registries support Docker Registry v3?

### DISCUSSION THREAD

**Discussion by amir-bialek** [argoproj-labs/argocd-image-updater#866] [General]
Hey, according to the documentation here:
https://argocd-image-updater.readthedocs.io/en/stable/configuration/registries/#supported-registries

Argocd Image Updater support 
```
Docker Registry v2 
```

However there is already a 3.0.0-beta.1 version. will it support v3 version soon?

**Comment by chengfang** [comment]
v3 is not widely adopted yet so currently is not in plan for image updater. However, we'll continue to check community interest and usage for potential feature additions.

**Reply by amir-bialek** [reply]
I am actually using it now.. it is working..
